package.json

```diff
@@ -30,6 +30,7 @@
   ],
   "dependencies": {
     "babel-plugin-transform-object-rest-spread": "^6.23.0",
+    "classnames": "^2.2.5",
```

Comment:
We already have a local version of classNames in utils.  We're not looking to add any dependencies here so can you remove this dependency.
But that's also alerted me to `babel-plugin-transform-object-rest-spread`
Could you move that to `devDependencies` please?

Reply:
Will do

Reply:
done